<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QWebView" name="webView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>771</width>
        <height>401</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QLabel" name="curLatLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>106</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Latitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="curLat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>29.7198</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>150</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination Latitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="destLat">
      <property name="geometry">
       <rect>
        <x>408</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>29.732395</string>
      </property>
     </widget>
     <widget class="QLabel" name="curLongLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>106</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Longitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="curLong">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-95.398087</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>150</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination Longitude</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="destLong">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>50</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>-95.394824</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateDest">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>161</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Destination</string>
      </property>
     </widget>
     <widget class="QPushButton" name="revertChanges">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>50</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Revert Changes</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
